tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": "src",
     "outDir": "build/dist",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
Why did this change? It wasn't originally part of the PR.

Reply:
I don't know, but this is the second time I've reverted it. Weird. Thanks for catching.